haiku-apps/bemobile/bemobile-0.9.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="A program to manage contacts and messages stored on your mobile phone"
```

Comment:
small change to make it shorter:
"Manage contacts and messages stored on a mobile phone"